src/main/webapp/app/overview/course-exercises/course-exercises.component.ts

```diff
@@ -218,6 +222,15 @@ export class CourseExercisesComponent implements OnInit, OnChanges, OnDestroy, A
         return !this.activeFilters.has(ExerciseFilter.UNRELEASED) || (exercise as QuizExercise)?.visibleToStudents;
     }
 
+    /**
+     * Method is called when enter key is pressed on search input
+     */
+    onSearch() {
```

Comment:
❌ For my this is NOT triggered when I press enter key while the focus is on the search field as the comment above says

Reply:
I updated the comment and added the trigger when enter key is pressed.